CMakeLists.txt

```
@@ -4,19 +4,18 @@ project (camp
   LANGUAGES CXX C
   VERSION 0.1.0)
 
-# Default to C++11 if not set so GTest/GMock can build
+# Default to C++17 if not set so GTest/GMock can build
 if (NOT BLT_CXX_STD)
-    set(BLT_CXX_STD "c++14" CACHE STRING "")
+    set(BLT_CXX_STD "c++17" CACHE STRING "")
 else()
-    set(_unsupported_cxx "c++98" "c++11")
+    set(_unsupported_cxx "c++98" "c++11" "c++14")
     if (BLT_CXX_STD IN_LIST _unsupported_cxx)
-      message(FATAL_ERROR "CAMP and the RAJA framework no longer support c++11, select a c++ standard of 14 or higher")
+      message(FATAL_ERROR "CAMP and the RAJA framework no longer support c++ standards below c++17, select a c++ standard of 14 or higher")
```

This message is contradictory and should be reworded.
```suggestion
      message(FATAL_ERROR "CAMP and the RAJA framework no longer support c++ standards below c++17, select a c++ standard of 17 or higher")
```